.github/contributing.md

```diff
@@ -1,7 +1,19 @@
 # Contributing to Skript
-Skript is an open source project. That basically means that you are
-encouraged to contribute by reporting issues and writing code. Basic
-guidelines are posted below to save time from everyone.
+Skript is an open source project, and you're encouraged to contribute to it.
+Both reporting issues and writing code help us. However, please review the
+following guidelines *before* doing either of these. Properly created issues
+and pull requests are often resolved faster than those that ignore them.
+
+## Behavior
+Please treat others with respect in the issue tracker and comments of pull
+requests. We hope that you are a decent person and do this without telling.
+Failing that, inappropriate behavior (and issues where it is observed) may
+be ignored, closed or even deleted. Repeated or particularly egregious behavior
```

Comment:
Doesn't this sound a little weird due to the "and issues where it is observed" being in brackets? Because if you ignore the brackets it can read "Failing that, inappropriate behavior may be closed", which probably doesn't make much sense. Or maybe that isn't a problem really, you can decide ;)

Reply:
Fixed in next commit